src/models/domain/passwordGeneratorPolicyOptions.ts

```diff
@@ -0,0 +1,50 @@
+import Domain from './domainBase';
+
+export class PasswordGeneratorPolicyOptions extends Domain {
+    minLength: number = 0;
+    useUppercase: boolean = false;
+    useLowercase: boolean = false;
+    useNumbers: boolean = false;
+    numberCount: number = 0;
+    useSpecial: boolean = false;
+    specialCount: number = 0;
+
+    constructor(
```

Comment:
Let's just ditch this ctor. Extra code that isn't being used. It's just more to maintain.

Reply:
Removed in next update